Src/main.c

```diff
@@ -106,10 +106,21 @@ int main(void)
 
   /* Infinite loop */
   /* USER CODE BEGIN WHILE */
+  //set servo PWM output relative to input voltage from potentiometer.
+  //potentiometer voltage from 0-3.3V, PWM modulation from 1-2ms on-time at 50Hz (period = 20ms)
   while (1)
   {
+      int tim16 = __HAL_TIM_GET_COUNTER(&htim16);
+      //ADC conversion: 1023 and first 3.3 are from ADC, last 3.3 and +1 from conversion to On-time.
+      double OnTime = ((HAL_ADC_GetValue(&hadc) / (1023 / 3.3)) / 3.3) + 1; //1 being the 0 degree servo position, divide input by 3.3 to get percent, 
```

Comment:
Keep in mind you need to init the ADC before calling HAL_ADC_GetValue :)

Reply:
Also, can you please explain your math? I'm a little confused by what you're trying to do here

Reply:
My math is rather unclear, I found the value that the ADC conversion outputs and am finding the input voltage. Then dividing that by the max voltage 3.3V and adding 1 as the on cycle is between 1ms nd 2ms for the full throw. 
I don't believe that I will keep that after switching to polling and interrupts. If I do, I will make it clearer.

Reply:
I see! One thing I recommend you to do is to use constants in your code which improves modifiability and readability